mapbox_gl_platform_interface/pubspec.yaml

```diff
@@ -10,4 +10,5 @@ dependencies:
 
 environment:
   sdk: '>=2.12.0 <3.0.0'
-  flutter: ">=2.0.0"
+  flutter: ">=3.0.0"
```

Comment:
Should this also apply to other modules like mapbox_gl_web and the other one?
Also, should we use the new dart here too?